### ISSUE TITLE
Do blocks mysteriously flagged

### ISSUE THREAD

**Issue by BigMcLargeHuge** [AtomLinter/linter-erb#55]
Hello,

I'm finding that the erb helper is mysteriously flagging the first line of several do blocks in my code.

For example, the first line of this block is flagged with the error `unexpected end-of-input, expecting ')'`

```
<%= link_to company_issues_path(@company), no_turbolink: true do %>
    <% if @company.issues.where(resolved: false).count == 0 %>
        <span id="warn">!</span>
    <% else %>
        <span id="warn" class="warning">!</span>
    <% end %>
    <span class="label"><%= @company.issues.where(resolved: false).count %> Unresolved</span>
<% end %>
```

Another example with error `unexpected ')'`:

```
<%= link_to company_path(@company.id), class: "link dashboard", data: {no_turbolink: true}, title: "Dashboard" do %>
    <span class="icon glyphicon glyphicon-dashboard"></span>
    <span class="label">Dashboard</span>
<% end %>
```

It's entirely possible that I might not be using the best syntax here so if there's a better way to write it feel free to make a suggestion.

**Comment by ricardograca** [comment]
Do you have `linter-htmlhint` installed as well?

**Comment by augustosamame** [comment]
I was having a similar issue. My ERB do blocks were flagged as ERB errors with `unexpected_keyword_end`

```
<%- @productattributes.where(name:'Imagen').each do |prodattrib| %>
  <div class="c-zoom">
    <%= image_tag (prodattrib.value) %>
  </div>
<%- end %>
```

I was not using linter-htmlhint, but upgrading my base Linter package version to version 1.11.8 solved the problem.

**Comment by Arcanemagus** [comment]
@augustosamame see https://github.com/AtomLinter/linter-erb/issues/54, @BigMcLargeHuge it's also possible you need to change your trim mode?

This entire "linter" is a hack since all it is doing is having `erb` generate Ruby code, and then running `ruby -wc` on the result...

**Comment by augustosamame** [comment]
Changing trim mode to '-' fixed the issues for if end. Thanks!

**Comment by UtamaruMonster** [comment]
Changing to trim - did not help same problem.

**Comment by ricardograca** [comment]
Then it isn't the same problem.

I should look into getting a new release out, but some changes in atom-linter are sb-exec have broken some tests recently and I'm afraid that might present more serious problems for some users.

**Comment by Arcanemagus** [comment]
Pushed out a new version, are any of you still seeing this after updating to v1.0.3?

**Comment by ricardograca** [comment]
This should be fixed since version 1.0.3. Let me know if you think there is any reason to reopen this issue.
